packages/dapp-svelte-wallet/api/src/wallet.js

```diff
@@ -388,7 +404,69 @@ export function buildRootObject(_vatPowers) {
               );
             }
 
-            switch (type) {
+            switch (obj.type) {
+              case 'walletRendezvous': {
+                const { dappAddresses } = obj;
+                const addressToWalletPK = makeStore('address');
+
+                // Create a rendezvous from all the dapp addresses to the wallet
+                // bridge facet.
+                const entries = dappAddresses.map(addr => {
+                  const walletPK = makePromiseKit();
+                  addressToWalletPK.init(addr, walletPK);
+                  return [addr, walletPK.promise];
+                });
+                const { notifier, completer } = E.G(
+                  E(rendezvous).startRendezvous(Object.fromEntries(entries)),
+                );
+
```

Comment:
It looks like in nearly all actually usages, we start from a single address or an array of addreses. Given that usage and that there might not be a `result` associated with an address, what about changing the API to take an array of addresses and an optional second array of results. It's not pretty, but having to do `Object.fromEntries` on every usage isn't pretty either.

Reply:
I've added this to the API, as well as a single-address form.